Allow installation of plugins not only in ~/.config

Used by app-misc/openrgb-plugin-* packages

diff --git a/PluginManager.cpp b/PluginManager.cpp
index b424b8b..991fb75 100644
--- a/PluginManager.cpp
+++ b/PluginManager.cpp
@@ -15,7 +15,8 @@ void PluginManager::ScanAndLoadPlugins(bool dark_theme)
     \*--------------------------------------------------------------------------------------*/
     OpenRGBPluginInterface *OpenRGBPlugin = nullptr;
 
-    const QDir pluginsDir = QString().fromStdString(ResourceManager::get()->GetConfigurationDirectory()) + "plugins/";
+    for (const QDir pluginsDir : {QString().fromStdString(ResourceManager::get()->GetConfigurationDirectory()) + "plugins/",
+                                  QString().fromStdString(GENTOO_PLUGINS_DIR)}) {
 
     std::vector<std::string> FileList;
 
@@ -54,4 +56,5 @@ void PluginManager::ScanAndLoadPlugins(bool dark_theme)
             std::cout << loader.errorString().toStdString() << std::endl;
         }
     }
+    }
 }
